brainiak/fcma/voxelselector.py

```diff
@@ -73,6 +73,10 @@ class VoxelSelector:
     voxel_unit: int, default 64
         The number of voxels assigned to a worker each time
 
+    process_num: Optional[int]
+        The number of processes used in cross validation.
+        If None, it will use up all available cores.
+
```

Comment:
We could use more precise language, e.g., "If None, the number of processes will equal the number of CPU cores."